lms/templates/certificates/valid.html

```diff
@@ -8,8 +8,8 @@
 
 <%include file="_accomplishment-rendering.html" />
 <div class="wrapper-about">
-    <div class="about">
+    <aside role="complimentary" class="about" aria-label="About edX Certificates">
```

Comment:
your intent here was good, but if you really want this to work, its going to have to be spelled like this: complementary ;)